2005-04-20  Neal H. Walfield  <neal@gnu.org>

        * sysdeps/generic/pt-setschedparam.c: policy's type is int, not
        int *.
        * include/pthread/pthread.h: Likewise.
        Reported by Christopher `Physicman' Bodenstein <cb@physicman.net>.

Index: libpthread/sysdeps/generic/pt-setschedparam.c
===================================================================
RCS file: /cvsroot/hurd/hurd/libpthread/sysdeps/generic/pt-setschedparam.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- libpthread/sysdeps/generic/pt-setschedparam.c	10 Oct 2002 23:05:06 -0000	1.1
+++ libpthread/sysdeps/generic/pt-setschedparam.c	20 Apr 2005 15:26:52 -0000	1.2
@@ -1,5 +1,5 @@
 /* Set the scheduling parameters for a thread.  Generic version.
-   Copyright (C) 2002 Free Software Foundation, Inc.
+   Copyright (C) 2002, 2005 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -21,7 +21,7 @@
 #include <pt-internal.h>
 
 int
-pthread_setschedparam (pthread_t thread, int *policy,
+pthread_setschedparam (pthread_t thread, int policy,
 		       const struct sched_param *param)
 {
   return ENOSYS;
Index: libpthread/include/pthread/pthread.h
===================================================================
RCS file: /cvsroot/hurd/hurd/libpthread/include/pthread/pthread.h,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libpthread/include/pthread/pthread.h	18 Jan 2005 09:57:35 -0000	1.3
+++ libpthread/include/pthread/pthread.h	20 Apr 2005 15:26:52 -0000	1.4
@@ -680,7 +680,7 @@
 				  struct sched_param *param);
 
 /* Set thread THREAD's scheduling paramters.  */
-extern int pthread_setschedparam (pthread_t thread, int *policy,
+extern int pthread_setschedparam (pthread_t thread, int policy,
 				  const struct sched_param *param);
 
 /* Set thread THREAD's scheduling priority.  */
